myhpi/search/templates/search/search_result.html

```diff
@@ -0,0 +1,15 @@
+{% load static wagtailcore_tags %}
+{% load bootstrap_icons %}
+
+<a href="{% pageurl result %}">
+    <div class="search-result">
+        <h5>{{ result.title }}</h5>
+        <span class="text-dark">
+            {% if result.search_description %}
+            {{ result.search_description }}
+            {% else %}
+            {{ result.specific.body|truncatewords:20 }}
+            {% endif %}
+        </span>
```

Comment:
This does not work for minutes as the field on minutes is called "text" (See https://github.com/fsr-de/myHPI/issues/206).

```suggestion
        <span class="text-dark">
            {% if result.search_description %}
                {{ result.search_description }}
            {% else %}
                {% if result.specific.text %}
                    {{ result.specific.text|truncatewords:20 }}
                {% else %}
                    {{ result.specific.body|truncatewords:20 }}
                {% endif %}
            {% endif %}
        </span>
```